matcher/service/openreview_interface.py

```diff
@@ -27,16 +25,42 @@ def __init__(self, client, config_note_id, logger=logging.getLogger(__name__)):
         self._demands = None
         self._constraints = None
 
-        self.validate_score_spec()
-
-    def validate_score_spec(self):
+        self._init_config_attributes(config_note_id)
+        self._validate_score_spec()
+
+    def _init_config_attributes(self, config_note_id):
+        try:
+            self.logger.debug('GET note id={}'.format(config_note_id))
+            self.config_note = self.client.get_note(config_note_id)
+        except openreview.OpenReviewException as error_handle:
+            self.logger.error('Config note note found: {}'.format(config_note_id))
+            raise ConfigNoteInterfaceError('Config note not found') from error_handle
```

Comment:
I don't think we need to wrap the exception in a new one, are you adding more information or behavior to the new exception?

Reply:
Fixed